vision_agent_tools/models/model_registry.py

```diff
@@ -0,0 +1,19 @@
+from typing import Dict, Callable
+from vision_agent_tools.shared_types import BaseMLModel
+from vision_agent_tools.models.florencev2 import Florencev2
+
+
+MODEL_REGISTRY: Dict[str, Callable[[], BaseMLModel]] = {"florence2": Florencev2}
```

Comment:
@hugohonda I think we should include all the models in the `MODEL_REGISTRY`

Reply:
I agree. But let's just keep it like this to merge this PR and we can add them in an upcoming PR.

Reply:
Ok! Will be added on the upcoming code